src/s1/cmp.js

```diff
@@ -181,25 +187,25 @@ const handleConsentResult = ({
 			warningMsg
 		};
 
-		callback.call(this, consent);
-
 		if (created && hasConsented !== hasConsentedCookie && !errorMsg) {
 			cmp.notify('onConsentChanged', consent);
 		}
+
```

Comment:
Moved this down below event notification to avoid unintentionally double calling on autoConsent flow